FreeRTOS/FreeRTOS-Plus-TCP
# Complete parser coverage

UnitTest: cover FreeRTOS_DNS_FreeRTOS

Description
-----------
UnitTest: cover FreeRTOS_DNS_FreeRTOS.c branches at 100%
Remove dead code.


Test Steps
-----------
Unit Tests pass with code coverage at 100%


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
